recipe/patches/0017-Fix-TZPATH-on-windows.patch

```diff
@@ -0,0 +1,24 @@
+From 5d81879c38e7c3f10e4c63636f17cc36fb170980 Mon Sep 17 00:00:00 2001
+From: Charles Bousseau <cbousseau@anaconda.com>
```

Comment:
?

Reply:
My start was to copy over the patches from Anaconda's 3.12 effort. It seems like they have changed the attribution on some of the patches. I can revert this here later.